src/store/modules/images.js

```diff
@@ -541,6 +590,11 @@ const actions = {
     commit('setCurrentImage', first_image)
   },
 
+  set_grouped_images ({ commit, state }) {
+    const grouped_image = state.grouped_images
```

Comment:
for consistency can we name `grouped_image` `grouped_images` instead?

Reply:
Lovely suggestion, done!